packages/fxa-settings/src/components/Settings/FlowRecoveryKeyHint/en.ftl

```diff
@@ -6,7 +6,8 @@
 # "key" here refers to the "account recovery key"
 flow-recovery-key-hint-header-v2 = Add a hint to help find your key
 # This message explains why saving a storage hint can be helpful. The account recovery key could be "stored" in a physical (e.g., printed) or virtual location (e.g., in a device folder or in the cloud).
-flow-recovery-key-hint-message-v2 = This hint should help you remember where you stored your account recovery key. We’ll show it to you when you use it to recover your data.
+# "it" here refers to the storage hint, NOT the "account recovery key"
+flow-recovery-key-hint-message-v3 = This hint should help you remember where you stored your account recovery key. We can show it to you during the password reset to recover your data.
```

Comment:
Was this new copy approved by Vesta / Design? Not sure if she would prefer "We'll show it" (or "we will") instead of "We can show it".

Reply:
Reverting to this string with "it" was suggested by Bryan, and reviewed and approved by Melissa (CX). Less repetitive than having "this hint" in two successive sentences.